### PR TITLE
Stream worker/node logs to manager with zeromq handlers/listeners. Fixes #135

### LINKED ISSUE #135: Stream/Collect Worker (and their Nodes) logs in their respective logs directory
Now that #113 adds support for adding zmq PUSH/PULL log handlers, it should be straight forward to collect these logs in their respective directory. Logs can be collected as they come.

### PR DESCRIPTION
**Pull request by umesh-timalsina**
Needs a rebase after #137.

ToDo:

- [x] Testing
- [x] Rebasing
- [x] Cleanups / More tests

### CODE REVIEW

**Comment by umesh-timalsina** [comment]
This should be good to go.

**Review by edavalosanaya — APPROVED** [review]
LGTM!

**Review by edavalosanaya — APPROVED** [review]
LGTM!

File: chimerapy/logger/common.py [hunk 1]
```diff
@@ -1,21 +1,51 @@
 import logging
-from logging import Formatter, StreamHandler, Filter
+from datetime import datetime
+from logging import Filter, Formatter, Handler, StreamHandler
+from logging.handlers import RotatingFileHandler
+from pathlib import Path
+
+MAX_BYTES_PER_FILE = 100 * 1024 * 1024  # 100MB
```
[the remark this answers is not in the document]
Comment (umesh-timalsina) [reply chimerapy/logger/common.py:8]: Addressed

**Pull request merged**